src/llmq/signing_shares.cpp

```diff
@@ -215,7 +216,8 @@ void CSigSharesManager::InterruptWorkerThread()
     workInterrupt();
 }
 
-void CSigSharesManager::ProcessMessage(const CNode& pfrom, const CSporkManager& sporkman, const std::string& msg_type, CDataStream& vRecv)
+void CSigSharesManager::ProcessMessage(const CNode& pfrom, PeerManager& peerman, const CSporkManager& sporkman,
```

Comment:
Had maintained arguments like that because all `ProcessMessage`s seem to start with `pfrom` and end with `msg_type, vRecv` ([source](https://github.com/dashpay/dash/blob/1e55310232dc000570438dc5fe981326b695fd79/src/net_processing.cpp#L5253-L5268)).